api/src/main/java/marquez/api/LineageResource.java

```diff
@@ -32,7 +32,7 @@ public LineageResource(@NonNull final ServiceFactory serviceFactory) {
   @ResponseMetered
   @ExceptionMetered
   @GET
-  @Path("/lineage")
+  @Path("{version:v1-beta|v1}/lineage")
   @Consumes(APPLICATION_JSON)
   @Produces(APPLICATION_JSON)
   public Response getLineage(
```

Comment:
I think we discussed moving this whole method to the `OpenLineageResource`. I think it's fine to update the `@Path` annotations to support the `v1-beta` path as well as the `v1` path for now. When we remove the deprecated path, we can consolidate the `@Path` annotations again.

Reply:
ok, sure

Reply:
Once you move the method `LineageResource.getLineage()` to `OpenLineageResource`, you can delete the class `LineageResource`.